relayer/client.go

```diff
@@ -144,9 +144,9 @@ func CreateClient(ctx context.Context, src, dst *Chain, srcUpdateHeader, dstUpda
 	}
 
 	// Create the ClientState we want on 'src' tracking 'dst'
```

Comment:
Does this comment need updated?

Reply:
It's still applicable but could be improved a bit. The code before these changes was incorrect and only worked because there was the safe assumption that both the src and dst chains were using a Tendermint light client.